jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/writer/Utf8Writer.java

```diff
@@ -11,37 +11,36 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.ee9.nested;
+package org.eclipse.jetty.io.writer;
 
 import java.io.IOException;
+import java.io.OutputStream;
 
 /**
- * OutputWriter.
- * A writer that can wrap a {@link HttpOutput} stream and provide
- * character encodings.
- *
+ * An implementation of {@link AbstractOutputStreamWriter} for
+ * an optimal UTF-8 conversion.
  * The UTF-8 encoding is done by this class and no additional
  * buffers or Writers are used.
- * The UTF-8 code was inspired by http://javolution.org
+ * The UTF-8 code was inspired by <a href="http://javolution.org">...</a>
  */
-public class Utf8HttpWriter extends HttpWriter
+public class Utf8Writer extends AbstractOutputStreamWriter
```

Comment:
This class should be internal or package local, or an private inner class of `AbstractOutputStreamWriter`.